### PR TITLE
Add new roles in alphabetical order in CustomRole Enum file

### PR DESCRIPTION
**Pull request by GameTechGuides**
Add new roles in alphabetical order in the `CustomRole` Enum file in `main.cs`.

* Add new roles such as `Admirer`, `Alchemist`, `Addict`, etc. (all of them), in alphabetical order.
* Ensure roles are listed alphabetically within their respective categories.
* Maintain the grouping of roles by categories such as Crewmate, Impostor, Neutral, etc.

### CODE REVIEW

**Comment by GameTechGuides** [comment]
Guys, sometimes I miss things. If I forget to add something in alphabetical order, please let me know.

**Comment by Ultradragon005** [comment]
I mean it rlly dosen't matter, ryuk just forgot to remove his comment

**Comment by KingPanda360** [comment]
Specter is renamed TOHE Phantom before v2024.6.18 came out and added the new vanilla role called "Phantom". It was renamed to Spectre because there would be two roles of the same name. This should not be removed from the list.

**Comment by Ultradragon005** [comment]
> Specter is renamed TOHE Phantom before v2024.6.18 came out and added the new vanilla role called "Phantom". It was renamed to Spectre because there would be two roles of the same name. This should not be removed from the list.

Oh wait I just realized... wtf... 💀 how old must this branch be.

Also it takes 2 seconds to do this with chatgpt if  we would like to

File: main.cs [hunk 1]
```diff
@@ -746,14 +745,13 @@ public enum CustomRoles
     Opportunist,
     Pelican,
     Pestilence,
-    Specter,
```
Comment (Tommy-XL) [inline main.cs:749]: Specter missed

Comment (GameTechGuides) [reply main.cs:749]: @Tommy-XL, thanks for pointing out that Spectre was missed. I'll correct that immediately.

**Comment by Ultradragon005** [comment]
[PDFen (1).pdf](https://github.com/user-attachments/files/16287782/PDFen.1.pdf)

The positions don't rlly matter, but if doing the whole thing might aswell go all out, so by asking gpt I got the exact positions (which take into account their second-msot, third-most fourth-msot letter etc..). (left side the current position, right side the most correct position)

**Comment by Pietrodjaowjao** [comment]
when this pr is merged i will create one randomizing the order completely

**Comment by Ultradragon005** [comment]
> when this pr is merged i will create one randomizing the order completely

yoooooo

**Pull request merged**